en/08_Changelogs/6.0.0.md

```diff
@@ -0,0 +1,23 @@
+---
+title: 6.0.0 (unreleased)
+---
+
+# 6.0.0 (unreleased)
+
+## Overview
+
+- [Run CanonicalURLMiddleware in all environment by default](#url-middleware)
+- [Bug fixes](#bug-fixes)
+
+## Run CanonicalURLMiddleware in all environment by default (#url-middleware)
+
+In Silvestripe 5.0 `CanonicalURLMiddleware` only runs in production by default. This lead to issues with `fetch` and APIs behaving differently in production environments to development. Silverstripe 6.0 changes this default to run the rules in dev, test and live by default.
```

Comment:
```suggestion
In Silverstripe CMS 5 the [`CanonicalURLMiddleware`](api:SilverStripe\Control\Middleware\CanonicalURLMiddleware) only runs in production by default. This lead to issues with `fetch` and APIs behaving differently in production environments to development. Silverstripe 6.0 changes this default to run the rules in `dev`, `test`, and `live` by default.
```
Please also include a short yaml snippet showing how to change this (or link to existing docs which show it)